test-e2e/manager-invite.js

```diff
@@ -123,7 +123,7 @@ skip("member can't invite", async (t) => {
   const creatorProject = await creator.getProject(createdProjectId)
 
   const responsePromise = creatorProject.$member.invite(member.deviceId, {
-    roleId: MEMBER_ROLE_ID,
+    roleId: COORDINATOR_ROLE_ID,
```

Comment:
Why has this test been changed? It's changed what it's testing. I know it's a skipped test for now, but the change could get lost down the line.

Reply:
Yeah, thanks for catching that, its a typo